novuRajan/gannt-chart
# handled-dependency-line-from-task-down-to-up

**Description**

This pull request resolves the issue related to the rendering of relationship lines between dependent tasks where blocker task is below the blocked task in the Gantt chart. The changes made ensure that the lines and arrowheads are dynamic based on the nature of the relationship

**Changes**

Improved logic to handle arrow direction based on task dependencies.
Added helper functions createSvgLine and createArrowhead.

**Testing instructions**

Open the Gantt chart, edit one of the tasks to have dependency in both the direction, and confirm that the relationship lines with the arrowheads are now rendered according to the dependency defined.
